subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/platforms.adoc

```diff
@@ -63,3 +414,43 @@ You can enable this behavior by using the `enforcedPlatform` keyword, instead of
 include::sample[dir="snippets/dependencyManagement/managingTransitiveDependencies-enforcedConstraintsFromBOM/groovy",files="build.gradle[tags=dependency-on-bom]"]
 include::sample[dir="snippets/dependencyManagement/managingTransitiveDependencies-enforcedConstraintsFromBOM/kotlin",files="build.gradle.kts[tags=dependency-on-bom]"]
 ====
+
+[[sub:platforms-vs-catalog]]
+== Should I use a platform or a catalog?
+
+Because platforms and catalogs both talk about dependency versions, can both be used to share dependency versions in a project, there might be a confusion regarding what to use and if one is preferable to the other.
+
+Ignoring the incubating state of version catalogs, the answer is that you should always use catalogs and _maybe_ use platforms in addition.
+
+A catalog helps with centralizing the dependency versions and is only, as it name implies, a catalog of dependencies you can pick from.
+We recommend using it to declare the coordinates of your dependencies, in all cases.
+It will be used by Gradle to generate type-safe accessors, present short-hand notations for external dependencies and it allows sharing easily those coordinates between different projects.
+Using a catalog will not have any kind of consequence on downstream consumers: it's transparent to them.
+
+A platform is a more heavyweight construct: it's a component of a dependency graph, like any other library.
+If you depend on a platform, that platform is itself a component in the graph.
+It means, in particular, that:
+
```

Comment:
For readability and consistency: I would make all bullet points Upper Case and end with a `.`. They are complete sentences.